src/iidm/BusBreakerVoltageLevel.hpp

```diff
@@ -66,9 +66,9 @@ class BusBreakerVoltageLevel : public VoltageLevel {
 
     Switch& addSwitch(std::unique_ptr<Switch>&& ptrSwitch, const std::string& busId1, const std::string& busId2);
 
-    void traverse(BusTerminal& terminal, voltage_level::TopologyTraverser& traverser) const;
+    void traverse(BusTerminal& terminal, Terminal::TopologyTraverser& traverser) const;
```

Comment:
All traverse method should return a boolean (see. https://github.com/powsybl/powsybl-core/pull/2044)